dll/win32/shell32/CDefaultContextMenu.cpp

```diff
@@ -90,6 +91,26 @@ UINT MapVerbToDfmCmd(_In_ LPCSTR verba)
     return 0;
 }
 
+static bool IsVerbListSeparator(int Ch)
+{
+    return Ch == ' ' || Ch == ','; // learn.microsoft.com/en-us/windows/win32/shell/context-menu-handlers
+}
+
+static int FindVerbInDefaultVerbList(LPCWSTR List, LPCWSTR Verb)
+{
+    for (UINT index = 0; *List; ++index)
+    {
+        while (IsVerbListSeparator(*List))
+            List++;
+        LPCWSTR Start = List;
+        while (*List && !IsVerbListSeparator(*List))
+            List++;
+        if (!_wcsnicmp(Verb, Start, List - Start) && List > Start)
```

Comment:
```suggestion
        if ((List > Start) && !_wcsnicmp(Verb, Start, List - Start))
```
so that in case it's <=, the wcsnicmp call doesn't get evaluated first...
(I should also note that List cannot be < Start)

Reply:
No, it is in the order I want. `List > Start` is only there to defend against the verb list string containing `,,` with no actual verb (meaning, the string is "empty"). The odds of this happening **and** the verb being empty is basically zero which is why I put that check after. I'll add a comment...